service/main.go

```diff
@@ -32,7 +32,7 @@ func init() {
 	flag.StringVar(&ramlFile, "ramlFile", "api.raml", "RAML file to parse")
 
 	serviceName = os.Getenv("SERVICE_NAME")
-	serviceRegistration = os.Getenv("SERVICE_REGISTRATION")
+	serviceRegistration = os.Getenv("SERVICE_REGISTRATION") == "1"
 	journal.Service = serviceName
```

Comment:
@buddhamagnet Not sure how the ```journal.Service``` works, but why assign a possible empty value to it? Condition to check for an empty ```serviceName``` happens below.